academy/3-my-own-chain/ignitecli.md

```diff
@@ -122,59 +128,162 @@ Available Commands:
   scaffold    Scaffold a new blockchain, module, message, query, and more
   chain       Build, initialize and start a blockchain node or perform other actions on the blockchain
   generate    Generate clients, API docs from source code
-  network     Launch a blockchain network in production
-  relayer     Connects blockchains via IBC protocol
+  account     Commands for managing accounts
+  relayer     Connect blockchains by using IBC protocol
   tools       Tools for advanced users
   docs        Show Ignite CLI docs
   version     Print the current build information
   help        Help about any command
+  completion  Generate the autocompletion script for the specified shell
 
 Flags:
   -h, --help   help for ignite
 
 Use "ignite [command] --help" for more information about a command.
 ```
 
+## Prepare Docker
+
+If you want portability and to avoid version issues, it is advisable to use [Docker](https://docs.docker.com/engine/install/). First, you need to create a `Dockerfile` that details the same preparation steps. Save this as `Dockerfile-ubuntu`:
+
+```dockerfile [https://github.com/cosmos/b9-checkers-academy-draft/blob/ignite-start/Dockerfile-ubuntu]
+FROM --platform=linux ubuntu:22.04
+ARG BUILDARCH
+
+# Change your versions here
+ENV GO_VERSION=1.18.3
```

Comment:
```suggestion
ENV GO_VERSION=1.18.5
```

Reply:
Do you suggest to change to keep the latest version?
The version stated here has been tested with the content, which is why I would keep it as it is until the next update.

Reply:
Sure! Go version upgrade are not breaking, though.